book/content/risk_assessment/risk_assessment.md

```diff
@@ -1,31 +1,43 @@
 # Risk Assessment - deciding how to manage your software
+## Prerequisites / recommended skill level
 
-## TL;DR
+| Prerequisite | Importance | Notes |
+| -------------|----------|------|
+|[Knowledge of Software Testing](https://www.test-institute.org/Introduction_To_Software_Testing.php) | Helpful | How it can be helpful in reducing the risk of failure |
```

Comment:
```suggestion
|[Knowledge of Software Testing](./testing/testing) | Helpful | How it can be helpful in reducing the risk of failure |
```

Reply:
Moreover @malvikasharan due to the previous commit  been done in outdated branch and which is before updation, the commit suggestion button is not activated  and that's why it shows to me that atleast 1 approving review is required by reviewers with write access and as I have commented below to do a new commit with enhanced summary only then I think it shows me the suggestions and all and not will add the tag for review changes.